projects/element-ng/modal/si-modal.service.ts

```diff
@@ -135,15 +135,10 @@ export class SiModalService {
       positionStrategy: this.overlay.position().global().centerHorizontally().centerVertically(),
       direction: isRTL() ? 'rtl' : 'ltr'
     });
-    return modalRef.overlayRef.attach(
-      new ComponentPortal(
-        SiModalComponent,
-        null,
-        injector,
-        null,
-        this.getContentProjectableNodes(injector, modalRef)
-      )
-    );
+    const compPortal = new ComponentPortal(SiModalComponent, null, injector);
+    // Ensure compatibility between Angular CDK 19 and 20 since constructor arguments have changed
+    compPortal.projectableNodes = this.getContentProjectableNodes(injector, modalRef);
```

Comment:
This should work at the moment as long as angular doesn't do any side effects related to `projectableNodes` within constructor itself. 
https://github.com/angular/components/blob/main/src/cdk/portal/portal.ts#L112